lib/screens/v2/candidate_generator/widgets/train_crowding.ex

```diff
@@ -56,7 +56,7 @@ defmodule Screens.V2.CandidateGenerator.Widgets.TrainCrowding do
            params |> Map.to_list() |> fetch_alerts_fn.() do
       next_train_prediction = List.first(predictions)
 
-      if logging_options.is_real_screen do
+      if not is_nil(next_train_prediction) and logging_options.is_real_screen do
```

Comment:
nit: I personally like `if next_train_prediction && logging_options.is_real_screen do` better for the brevity, but just personal preference. Not sure if using `is_nil/1` is more idiomatic though

Reply:
I could've sworn I tried that earlier, but the problem is I was using `and` instead of `&&`... Thanks for the suggestion cause I definitely like this better.